pkg/config/config_template.yaml

```diff
@@ -1282,6 +1282,11 @@ api_key:
   ## If you are sending OTel traces and do not want stats computed by span kind, you need to disable this flag and remove the "enable_otlp_compute_top_level_by_span_kind" APM feature if present.
   # compute_stats_by_span_kind: false
 
+  ## @param peer_service_aggregation - bool - default: false
+  ## @env DD_APM_PEER_SERVICE_AGGREGATION - bool - default: false
+  ## DEPRECATED - please use `peer_tags_aggregation` instead.
```

Comment:
```suggestion
  ## DEPRECATED - use `peer_tags_aggregation` instead.
```

Reply:
skipping this as this PR is simply a revert of another change